### ISSUE TITLE
Usage of angle vs. direction

### ISSUE THREAD

**Issue by zhardy-lanl** [Open-Sn/opensn#104]
Originated from discussion on #100.

**Comment by ragusa** [comment]
angle is a misnomer in most of the code. often, what is meant is a direction, which, as we all know, takes 2 angles to be fully defined in 2d and 3d. 
I am of the strong opinion that angles should become directions. My only (small) hesitation is angle_set versus direction_set because angle_set rolls better of the tongue.

**Comment by zhardy-lanl** [comment]
I tend to agree with this. Angle seems like a (theta, varphi) pair whereas direction seems like a vector. We could call it an `OrdinateSet` which sounds better then `DirectionSet` in my opinion.

**Comment by wdhawkins** [comment]
I agree. We've gotten in the habit of saying angles, angle sets, etc. when we really mean direction. I have no strong feeling s on nomenclature. I like Zach's `OrdinateSet` suggestion.
